### PR TITLE
Switch to flakes, attempt 2

### PR DESCRIPTION
**Pull request by balsoft**
The first attempt was met with very much deserved criticism and feedback; So, after a lot of discussion, this is the implementation of option 7 of our deploy system list.

### CODE REVIEW

**Comment by balsoft** [comment]
Switched to our new shiny flakes structure. Companion PRs for app repos will follow.

**Comment by balsoft** [comment]
- https://github.com/serokell/stakerdao-agora/pull/72
- https://github.com/StakerDAO/blend-app/pull/109
- https://github.com/StakerDAO/bridge-web/pull/44

File: profiles/bridge.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{ config, pkgs, inputs, ... }:
+let
+  profiles = "/nix/var/nix/profiles/per-user/deploy";
```
Comment (zhenyavinogradov) [inline profiles/bridge.nix:3]: Hm, there is no `deploy` user configured. Is by added in the app module?

**Review by zhenyavinogradov — COMMENTED** [review]
Looks good mostly

File: profiles/bridge.nix (hunk 1 above)
Comment (balsoft) [reply profiles/bridge.nix:3]: Now they are configured in all profiles.

File: flake.nix [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+        profiles.system.user = "root";
+        profiles.system.path =
+          deploy-rs.lib.${system}.activate.nixos nixosConfig;
+      }) self.nixosConfigurations;
+
+      devShell = mapAttrs (system: deploy:
+        let pkgs = nixpkgs.legacyPackages.${system}.extend serokell-nix.overlay;
+        in pkgs.mkShell {
+          buildInputs = [ deploy (terraformFor pkgs) pkgs.nixUnstable ];
```
[the remark this answers is not in the document]
Comment (balsoft) [reply flake.nix:76]: Yep, Chris needs it for development

File: profiles/agora.nix [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  profile-root = "/nix/var/nix/profiles/per-user/deploy";
+in {
+  imports = [
+    inputs.stakerdao-agora.nixosModules.combined
+  ];
+
+  users.users.deploy = {
+    isNormalUser = true;
+    openssh.authorizedKeys.keys = [
+      "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGItnQMIZO55M7hnlAcJ9z4a0IWHajZxd3cBrESR6HpN deploy"
```
[the remark this answers is not in the document]
Comment (balsoft) [reply profiles/agora.nix:17]: CIs for corresponding projects have those keys configured for them.

**Pull request merged**